tests/lua/uarch-riscv-tests.lua

```diff
@@ -389,18 +389,16 @@ local function write_access_to_log(access, out, indent, last)
     util.indentout(out, indent + 1, '"type": "%s",\n', access.type)
     util.indentout(out, indent + 1, '"address": %u,\n', access.address)
     util.indentout(out, indent + 1, '"log2_size": %u,\n', access.log2_size)
+    local read_value = '""' -- Solidity JSON parser breaks, if this field is null
+    if access.read then read_value = '"' .. util.hexstring(access.read) .. '"' end
+    util.indentout(out, indent + 1, '"read_value": %s,\n', read_value)
+    util.indentout(out, indent + 1, '"read_hash": "%s",\n', util.hexhash(access.read_hash))
     if access.type == "write" then
-        local value = "null"
-        if access.written then value = '"' .. util.hexstring(access.written) .. '"' end
```

Comment:
I see the written value is gone in the tests, the written value is not relevant anymore? Not even for debugging? (I imagine `written_hash` can fulfill it needs). Would make sense to have `written_value` here?

Reply:
I’d like to keep the written value just for debugging.